3 - daemon_set/ds_task.md

```diff
@@ -4,11 +4,18 @@
 __Task 4, step 5: Once you apply the DaemonSet, what happens and why?__
 
 ```
-enter answer here
+Creates Daemon set, but it is not available
```

Comment:
It creates the DaemonSet but no node labels match the node selector so no pods are scheduled.